Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -90,13 +89,21 @@ public virtual void RenderCustomInspector()
             bool isPlayMode = EditorApplication.isPlaying || EditorApplication.isPaused;
 
             #region General Settings
-            EditorGUILayout.BeginHorizontal();
+            using (new EditorGUILayout.HorizontalScope())
+            {
                 InspectorUIUtility.DrawTitle("General");
-                InspectorUIUtility.RenderDocLinkButton(Interactable_URL);
-            EditorGUILayout.EndHorizontal();
 
-            EditorGUILayout.BeginVertical(EditorStyles.helpBox);
+                if (target != null)
+                {
+                    var helpURL = target.GetType().GetCustomAttribute<HelpURLAttribute>();
+                    if (helpURL != null)
+                    {
+                        InspectorUIUtility.RenderDocumentationButton(helpURL.URL);
+                    }
+                }
+            }
 
+            EditorGUILayout.BeginVertical(EditorStyles.helpBox);
```

Comment:
why not use the same pattern as above here?

Reply:
Because I want #5493 to go through